DatadogInternal/Sources/NetworkInstrumentation/URLSession/URLSessionTaskInterception.swift

```diff
@@ -0,0 +1,231 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+
+public class URLSessionTaskInterception {
```

Comment:
Almost identical to the v1 `URLSessionTaskInterception`, only the span context property has been replaced by a `trace` ids.